pkg/network/tracer/tracer_usm_linux_test.go

```diff
@@ -1038,22 +1038,19 @@ func TestTLSClassification(t *testing.T) {
 		t.Skip("TLS classification platform not supported")
 	}
 
-	// testContext shares the context of a given test.
-	// It contains common variable used by all tests, and allows extending the context dynamically by setting more
-	type testContext struct{}
+	tr := setupTracer(t, cfg)
+
 	type tlsTest struct {
 		name            string
-		context         testContext
-		preTracerSetup  func(t *testing.T, ctx testContext)
-		postTracerSetup func(t *testing.T, ctx testContext)
-		validation      func(t *testing.T, ctx testContext, tr *Tracer)
-		teardown        func(t *testing.T, ctx testContext)
+		postTracerSetup func(t *testing.T)
+		validation      func(t *testing.T, tr *Tracer)
 	}
-	tests := []tlsTest{}
-	for _, tlsVersion := range []string{"-tls1", "-tls1_1", "-tls1_2", "-tls1_3"} {
+	scenarios := []string{"-tls1", "-tls1_1", "-tls1_2", "-tls1_3"}
```

Comment:
Maybe add to the name "TLS-" in order to avoid adding for each name of the scenarios the "-" in the begging

Reply:
tried to clarify. let me know if that addresses your comment